pkg/wsman/hash.go

```diff
@@ -52,8 +52,8 @@ func (c *authChallenge) hashURI(method, uri string) string {
 func (c *authChallenge) response(method, uri, cnonce string) (string, error) {
 	c.NonceCount++
 
-	if c.Qop == "auth" || c.Qop == "" {
-		if c.Qop == "auth" {
+	if strings.Contains(c.Qop, "auth") || strings.Contains(c.Qop, "auth-int") || c.Qop == "" {
+		if strings.Contains(c.Qop, "auth") || strings.Contains(c.Qop, "auth-int") {
```

Comment:
similarly, we already know this is true given the previous condition, the only possible condition where this is not true is when its blank, so change this to if c.Qop == ""

Reply:
removed auth-int check.  the c.Qop == "" was there from before so that is needed on the first call when the Authorization headers are blank